tests/Prep.m

```diff
@@ -0,0 +1,72 @@
+classdef Prep < matlab.unittest.TestCase
+    % Setup and teardown for tests.
+    properties (Constant)
+        CONN_INFO = struct(...
+            'host', dj.config('databaseHost'), ...
+            'user', dj.config('databaseUser'), ...
+            'password', dj.config('databasePassword'));
+        PREFIX = 'u19_test_';
+        test_root = dj.config('root_dir');
```

Comment:
This test makes an assumption that there is some preloading of `dj.config`. We should capture this within the test so that it is entirely self-contained.